core/src/main/java/kafka/server/share/SharePartitionManager.java

```diff
@@ -697,7 +696,6 @@ private SharePartitionKey sharePartitionKey(String groupId, TopicIdPartition top
      */
     // Visible for testing.
     long offsetForEarliestTimestamp(TopicIdPartition topicIdPartition) {
-        // TODO: We need to know the isolation level from group configs, for now we are passing Option.empty() for isolationLevel
```

Comment:
It might be useful to add a comment why the isolation level can be empty.

Reply:
Done.